### PR TITLE
Updated swupd.zsh to reflect changes

### PR DESCRIPTION
**Pull request by lebensterben**
zsh-completion updated in accordance to
https://github.com/clearlinux/swupd-client/commit/10bc311e4cc8a44b1a13ce99c5586ec4e453a0b5

### CODE REVIEW

**Comment by otaviobp** [comment]
Thanks and sorry for forgetting to update those.
@castulo Can you check if the flags are correct?

File: swupd.zsh [hunk 1]
```diff
@@ -157,7 +157,7 @@ if [[ -n "$state" ]]; then
         update)
           local -a updates; updates=(
             $global_opts
-            '(help status -d --download )'{-d,--download}'[Download all content, but do not actually install the update]'
+            '(help status -d --download )--download[Download all content, but do not actually install the update]'
```
Comment (castulo) [inline swupd.zsh]: I think you forgot to remove the '-d' from here: `(help status -d --download )`

Comment (lebensterben) [reply swupd.zsh]: Fixed. Thanks.

**Comment by lebensterben** [comment]
I found some other changes I need to make.

For example in one of the commit `swupd bundle add` has a new option to ignore optional package.

So please hold on and I will get this done later today.

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>

**Comment by castulo** [comment]
> I found some other changes I need to make.
> 
> For example in one of the commit `swupd bundle add` has a new option to ignore optional package.
> 
> So please hold on and I will get this done later today.
> 
> Sent with [GitHawk](http://githawk.com)

yes, also another PR just merged today too that adds a global option called `--no-progress`

**Pull request merged**